src/API/Google/Connection.php

```diff
@@ -129,6 +122,44 @@ public function get_status(): array {
 		}
 	}
 
+	/**
+	 * Get the reconnect status which checks:
+	 * - The Google account is connected
+	 * - We have access to the connected MC account
+	 * - We have access to the connected Ads account
+	 *
+	 * @return array
+	 * @throws Exception When a ClientException is caught or the response contains an error.
+	 */
+	public function get_reconnect_status(): array {
+		$status = $this->get_status();
+		$email  = $status['email'] ?: '';
+
+		if ( ! isset( $response['status'] ) || 'connected' !== $response['status'] ) {
```

Comment:
```json
{
  "email": "eason.su@a8c.com",
  "active":"yes"
}
```
I got above result when Google account is connected. Looks like the `$response` should be `$status`?

Reply:
Thanks for catching that, yes that's a copy paste error, it should be $status.